docs/api-docs/apps/guide/apps-12-requirements.md

```diff
@@ -28,6 +28,7 @@ Don't reference competitor platforms in the app's listing information or dashboa
 "Marketplace by BigCommerce" or "Search by BigCommerce").
 - Develop app client-side code to be compatible with all BigCommerce [supported browsers](https://support.bigcommerce.com/s/article/Themes-Supported-Browsers) and default features.
 - Include applicable test account details in your test instructions with the submission for any third-party services required in the app.
+- Use only BigCommerce API resources to perform relevant API functions.

```

Comment:
The "only" sort of gets me because at first glance it might look like you can't use other APIs in your app. I wasn't sure what "relevant API functions" meant precisely, so I looked at the ticket. Here is a suggestion: 

```suggestion
- Use BigCommerce APIs for app functionality wherever possible. For example, don't require a merchant to manually import a product CSV if those products can be retrieved using the BigCommerce API.

```